File: Easy/2026-09-02-3875-Construct-Uniform-Parity-Array-I/solution.java
class Solution {
    public boolean canConstructArray(int[] nums1) {
        boolean hasOdd = false;
        boolean hasEven = false;

        for (int num : nums1) {
            if (num % 2 == 0) {
                hasEven = true;
            } else {
                hasOdd = true;
            }
        }

        return !(hasOdd && hasEven);
    }
}

File: Easy/2026-09-02-3875-Construct-Uniform-Parity-Array-I/solution.py
class Solution:
    def canConstructArray(self, nums1: List[int]) -> bool:
        odd_count = sum(1 for num in nums1 if num % 2 != 0)
        even_count = len(nums1) - odd_count
        return odd_count == len(nums1) or even_count == len(nums1)